probe_scraper/scrapers/git_scraper.py

```diff
@@ -13,7 +13,17 @@
 
 MIN_DATES = {
     # Previous versions of the file were not schema-compatible
-    "glean": "2019-01-25 00:00:00"
+    "glean": "2019-03-15 00:00:00"
+}
+
+IGNORE_COMMITS = {
```

Comment:
This was required because these commits all came on the same day, so simply limiting by date wasn't enough. That should be the only case when `IGNORE_COMMITS` should be used and not `MIN_DATES`.